package.json

```diff
@@ -110,5 +112,6 @@
   },
   "tsd": {
     "directory": "./dist/types"
-  }
-}
\ No newline at end of file
+  },
+  "packageManager": "yarn@4.2.2"
+}
```

Comment:
This was suggested by [react-native upgrade helper](https://react-native-community.github.io/upgrade-helper/) for react-native 0.73 -> 0.74